src/backtracking/python/all_permutations/src/all_permutations.py

```diff
@@ -0,0 +1,14 @@
+def all_permutations(elements):
```

Comment:
The code looks pretty nice, and it passed the test successfully, you just need to fulfill the requirements that the checklist in your pull request mentioned, and then your code will be ready for merging!!

Reply:
Thank you! I've updated the README documentation

Reply:
Nice work!!